src/Surfnet/ServiceProviderDashboard/Application/Command/Entity/UpdateEntityAclCommand.php

```diff
@@ -26,65 +28,53 @@
 
 class UpdateEntityAclCommand implements Command
 {
-    private $manageEntity;
-
-    /**
-     * @var IdentityProvider[]
-     * @Assert\All({
-     *     @Assert\NotBlank(),
-     *     @Assert\Type(type="Surfnet\ServiceProviderDashboard\Domain\Entity\IdentityProvider")
-     * })
-     */
-    private $selected;
-
-    /**
-     * @var bool
-     */
-    private $selectAll = false;
-
-    public function __construct(ManageEntity $entity, array $selectedIdps, bool $selectAll)
-    {
-        $this->manageEntity = $entity;
-        $this->selected = $selectedIdps;
-        $this->selectAll = $selectAll;
+    public function __construct(
+        private readonly ManageEntity $manageEntity,
+        /**
+         * @var                                                                                 IdentityProvider[]
+         */
+        #[Assert\All([
+            new Assert\NotBlank(),
+            new Assert\Type(type: IdentityProvider::class),
+        ])]
+        private array $selected,
+        private bool $selectAll,
+    ) {
     }
 
-    /**
-     * @return string
-     */
-    public function getManageEntity()
+    public function getManageEntity(): ManageEntity
     {
         return $this->manageEntity;
     }
 
     /**
      * @return IdentityProvider[]
      */
-    public function getSelected()
+    public function getSelected(): array
     {
         return $this->selected;
     }
 
     /**
      * @param IdentityProvider[] $idps
      */
-    public function setSelected(array $idps)
+    public function setSelected(array $idps): void
     {
         $this->selected = $idps;
     }
 
     /**
      * @return bool
      */
```

Comment:
:wastebasket:

Reply:
done

Reply:
The PHPDoc is still there right? 

/Nitpick